package.json

```diff
@@ -65,7 +66,8 @@
     "prettier": "^2.1.2",
     "ts-node": "^9.1.1",
     "ttypescript": "^1.5.12",
-    "typescript": "4.1.3"
+    "type-coverage": "^2.17.2",
```

Comment:
why move to root?

Reply:
I don't know. type-coverage cannot run on my machine. It always reports

```
0 / 0 NaN%
type-coverage success.
```